roles/sap_general_preconfigure/tasks/RedHat/configuration.yml

```diff
@@ -35,7 +35,11 @@
   when: sap_general_preconfigure_create_directories or sap_general_preconfigure_modify_selinux_labels
 
 - name: Configure - Include configuration actions for required sapnotes
-  ansible.builtin.include_tasks: "sapnote/{{ sap_note_line_item.number }}.yml"
+  ansible.builtin.include_tasks:
+    file: "sapnote/{{ sap_note_line_item.number }}.yml"
   with_items: "{{ __sap_general_preconfigure_sapnotes_versions | difference(['']) }}"
   loop_control:
     loop_var: sap_note_line_item
+  tags:
+    - sap_general_preconfigure_configuration
+    - sap_general_preconfigure_configuration_all_steps
```

Comment:
Is the tag `sap_general_preconfigure_configuration_all_steps` really necessary?

Reply:
> Is the tag `sap_general_preconfigure_configuration_all_steps` really necessary?

Thanks! I tested again and it found no longer a combination of tags which would require both tags. So it seems it's really not required. I'll update the code accordingly.

Reply:
I was a bit too fast. Further testing revealed that only with both tags, `sap_general_preconfigure_configuration` and  `sap_general_preconfigure_configuration_all_steps`, it is possible to execute just one or a few of the configuration steps.
With just the tag `sap_general_preconfigure_configuration`, it is possible to run all configuration steps and skip some of them. But I want more. ;-)
See also the examples in README.md.